practice-two/src/styles/components/button-rectangle.css

```diff
@@ -1 +1,20 @@
+.btn-rectangle {
```

Comment:
button radius and reactangle have some things the same, you should define a common class for them to reduce duplicate code